docs/application/web/api/6.5/device_api/wearable/tizen/ml_single.html

```diff
@@ -750,6 +892,9 @@ <h2 id="full-webidl">3. Full WebIDL</h2>
   [Callback=FunctionOnly, NoInterfaceObject] interface OpenModelSuccessCallback {
     void onsuccess(<a href="#SingleShot">SingleShot</a> singleShot);
   };
+  [Callback=FunctionOnly, NoInterfaceObject] interface InvokeAsyncSuccessCallback {
+    void onsuccess(<a href="ml.html#TensorsData">TensorsData</a> tensorsData);
+  };
```

Comment:
I reviewed this file again to check for any broken links in the document. I found broken links in the following lines. Kindly check and update the links.

Line # 26, 28, 30, 410, 812, and 839